File: 2778-sum-of-squares-of-special-elements/2778-sum-of-squares-of-special-elements.cpp
class Solution {
public:
    int sumOfSquares(vector<int>& nums) {
        long long res=0;
        long long n=nums.size();
        for(long long i=1;i<n+1;i++){
            if(n%i==0){
                res+=nums[i-1]*nums[i-1];
            }
        }
        return res;
        
        
    }
};

File: 2778-sum-of-squares-of-special-elements/2778-sum-of-squares-of-special-elements.py
class Solution:
    def sumOfSquares(self, nums: List[int]) -> int:
        res=0
        n=len(nums)
        for i in range(1,len(nums)+1):
            if n%i==0:
                res+=nums[i-1]*nums[i-1]
        return res